schema/artwork/index.js

```diff
@@ -309,8 +309,37 @@ const ArtworkType = new GraphQLObjectType({
       availability: {
         type: GraphQLString,
       },
+      is_on_hold: {
+        type: GraphQLString,
+        resolve: ({ availability }) => availability === 'on hold',
+      },
+      is_not_for_sale: {
+        type: GraphQLString,
+        resolve: ({ sale_message }) => sale_message === 'Not for sale',
+      },
       sale_message: {
         type: GraphQLString,
+        args: {
+          format: {
+            type: GraphQLBoolean,
+            default: false,
+            description: 'Show a formatted response instead of the one that comes from gravity',
+          },
+        },
+        resolve: (artwork, { format }) => {
```

Comment:
I love this format, except that in the case of the artwork brick, it seems like we don't want to display anything if it's `Sold`.

However, looks like the artwork page (force + microg) still wants to display `Sold`. That _seems_ to be the only inconsistency that would prevent the artwork page from just using this directly.

But, it's still not a big deal, on artwork pages we can just display `sold if artwork.is_sold`, otherwise this.